src/plugins/cosmos/components/RewardsRow/RewardsRow.tsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, FlexProps } from '@chakra-ui/layout'
+import { Text as CText } from '@chakra-ui/react'
+import { Text } from 'components/Text'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type RewardsRowProps = {
+  assetSymbol: string
+  cryptoRewardsAmount: BigNumber
+  fiatRate: BigNumber
+}
+
+export const RewardsRow = ({
+  assetSymbol,
+  cryptoRewardsAmount,
+  fiatRate,
+  ...styleProps
+}: RewardsRowProps & FlexProps) => (
+  <Flex {...styleProps}>
+    <Flex width='50%' height='20px'>
+      <Text translation={'defi.rewards'} />
+    </Flex>
+    <Flex direction='column' alignItems='flex-end' width='100%'>
+      <CText fontWeight='semibold' color='green.500'>
+        {`$${cryptoRewardsAmount.times(fiatRate).toPrecision()}`}
```

Comment:
i think we have some sort of i18n version of `$` formatting - have a look at balances on the asset pages for how it's done so we get commas/periods correct for other locales

Reply:
or `<Amount.Fiat>` component?